src/getHumanAge.test.js

```diff
@@ -7,4 +7,36 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should discard the remainder', () => {
+    expect(getHumanAge(27, 27)).toEqual([2, 2]);
+  });
+
+  test('should return [0, 0] for catAge = 0, dogAge = 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] for catAge = 14, dogAge = 14', () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  test('should return [1, 1] for catAge = 15, dogAge = 15', () => {
```

Comment:
The test case seems to be incorrect. If catAge = 15 and dogAge = 15, the conversion to human years should probably not be [1, 1]. Please confirm with the conversion rules.